diff --git a/setup.py b/setup.py
index 2c881fc..41f37e1 100755
--- a/setup.py
+++ b/setup.py
@@ -22,8 +22,12 @@ setup(
     tests_require=tests_require,
     extras_require={"test": tests_require},
     packages=find_packages(exclude=["test"]),
-    scripts=glob.glob("scripts/*"),
     include_package_data=True,
+    entry_points = '''
+        [console_scripts]
+        yq=yq:main
+        xq=yq:xq_cli
+    ''',
     test_suite="test",
     classifiers=[
         "Intended Audience :: Developers",
diff --git a/yq/__init__.py b/yq/__init__.py
index 95fd1e8..0800401 100755
--- a/yq/__init__.py
+++ b/yq/__init__.py
@@ -82,6 +82,9 @@ for arg in jq_arg_spec:
 parser.add_argument("jq_filter")
 parser.add_argument("files", nargs="*", type=argparse.FileType())
 
+def xq_cli():
+    main(input_format="xml")
+
 def main(args=None, input_format="yaml"):
     args, jq_args = parser.parse_known_args(args=args)
     for arg in jq_arg_spec:
